lib/acme-client.rb

```diff
@@ -1,3 +1,7 @@
+# This file lives here for backward compatibility only and should be deprecated soon
+# gem named acme-client should be required as
```

Comment:
Bundler "knows" about namespaced libs and would autorequire acme/client without complaints if there wouldn't be acme-client.rb file.
https://github.com/bundler/bundler/blob/master/lib/bundler/runtime.rb#L97